### PR TITLE
[NEXUS-8813] Hookup basic functional-testsuite

### PR DESCRIPTION
**Pull request by jdillon**
https://issues.sonatype.org/browse/NEXUS-8813

related:

https://github.com/sonatype/nexus-pro/pull/564
https://github.com/sonatype/nexus-bundles/pull/156

### CODE REVIEW

File: testsuite/functional-testsuite/src/test/script/prepare_siestajs.groovy [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+// Ensure scripts and binaries are executable
+ant.chmod(perm: 'u+x') {
+  fileset(dir: "$siestaDir/bin") {
+    include(name: '**')
+  }
+}
+
+// cache extjs resources for siesta-js harness
+ant.get(src: 'http://cdn.sencha.com/ext/gpl/5.1.0/build/ext-all-debug.js',
+    dest: "$siestaDir/extjs-all.js")
```
Comment (kellyrob99) [inline testsuite/functional-testsuite/src/test/script/prepare_siestajs.groovy]: might be worthwhile to preserve the ext-all-debug.js name here and avoid any possible confusion for the future

**Comment by kellyrob99** [comment]
+1

**Comment by jdillon** [comment]
On hold for the moment to sort out some stability issues first.

**Comment by jdillon** [comment]
manually merged

**Pull request closed without merging**